site/gatsby-site/src/components/forms/TextInputGroup.js

```diff
@@ -122,24 +132,34 @@ const Input = ({
   className = '',
   addOnComponent,
   ...props
-}) => (
-  <div className="w-full flex">
-    <input
-      type={type}
-      name={name}
-      className={`bg-gray-50 border text-gray-900 text-sm rounded-lg block w-full p-2.5 dark:bg-gray-700 dark:placeholder-gray-400 dark:text-white ${
-        errors && touched && touched[name] && errors[name]
-          ? 'border-red-600 focus:ring-red-500'
-          : 'border-gray-300 dark:border-gray-600 dark:focus:border-blue-500 focus:border-blue-500 focus:ring-blue-500 dark:focus:ring-blue-500'
-      } ${className}`}
-      placeholder={placeholder}
-      value={values[name] || props.defaultValue || ''}
-      {...props}
-      onChange={handleChange}
-      onBlur={handleBlur}
-    />
-    {addOnComponent}
-  </div>
-);
+}) => {
+  const [inputName, setInputName] = useState('');
```

Comment:
This adds an extra render on every input. Why do you need it?

Reply:
Fixed!